File: Alura/Javascript/JavaScript para Back-End/Parte 3/Curso 2/Docente.js
import User from "./User.js";

export default class Docente extends User {
  constructor(nome, email, nascimento, role = "admin", ativo = true) {
    super(nome, email, nascimento, role);
  }

  aprovaEstudante(estudante,curso){
    return `Estudante ${estudante} passou no curso de ${curso}`;
  }
}

File: Alura/Java/Praticando Java/09 - Heranca, Polimorfismo e Interfaces/Docente.java
public class Docente extends Pessoa{
    String disciplina;

    public Docente(String nome, int idade,String disciplina) {
        super(nome, idade);
        this.disciplina = disciplina;
    }
    @Override
    public void exibirDados() {
        System.out.printf("Docente: %s - Idade: %d - Disciplina: %s\n", nome,idade,disciplina);
    }
}
